src/plugins/elasticsearch/index.js

```diff
@@ -28,8 +28,8 @@ module.exports = function ({ Plugin }) {
           cert: string(),
           key: string()
         }).default(),
-        apiVersion: string().default('2.0'),
-        engineVersion: string().valid('^2.1.0').default('^2.1.0')
+        apiVersion: Joi.string().default('master'),
```

Comment:
Maybe I missed a conversation about how all this is going to be handled. Is the plan to replace master everywhere with the correct version of ES when we cut a release?